### PR TITLE
JSDocs for list view checklist

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-view-checklist?_sp.spa=pr-jsdocs-090220](https://developer.blackbaud.com/skyux-list-builder-view-checklist?_sp.spa=pr-jsdocs-090220)

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts [hunk 1]
```diff
@@ -78,22 +78,46 @@ import {
...
+   * The default function searches view data on label, description,
+   * and category if field selectors are defined for the given field.
+   * @default default search function
+   */
   @Input()
   public search: (data: any, searchText: string) => boolean = this.searchFunction();
 
+  /**
+   * Specifies the name of the label field selector.
+   * @default label
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]:
```suggestion
   * @default 'label'
```

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts [hunk 2]
```diff
@@ -78,22 +78,46 @@ import {
...
+   */
   /* tslint:disable */
   @Input('label')
   public labelFieldSelector: string = 'label';
   /* tslint:enable */
 
+  /**
+   * Specifies the name of the description field selector.
+   * @default description
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]:
```suggestion
   * @default 'description'
```

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts [hunk 3]
```diff
@@ -78,22 +78,46 @@ import {
...
+   */
   @Input()
   public description: string = 'description';
 
+  /**
+   * Specifies how many items users can select.
+   * `single` allows users to select one item in the checklist, while `multiple`
+   * allows users to select multiple items in the checklist.
+   * @default multiple
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]:
```suggestion
   * @default 'multiple'
```

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]: Updated.

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]: Updated.

File: src/app/public/modules/list-view-checklist/list-view-checklist.component.ts (hunk 3 above)
Comment (blackbaud-johnly) [reply src/app/public/modules/list-view-checklist/list-view-checklist.component.ts]: Updated

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-list-builder-view-checklist?_sp.spa=pr-jsdocs-090220](https://developer.blackbaud.com/skyux-list-builder-view-checklist?_sp.spa=pr-jsdocs-090220)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**